pkg/users/group-details.jsx

```diff
@@ -0,0 +1,187 @@
+                .then(() => {
+                    setSelected([...selected, account]);
+                    setIsOpenAccount(false);
+                }, show_unexpected_error);
+    };
+
+    const onSelectAccount = (event, selection) => {
+        if (selected.includes(selection)) {
+            removeAccount(selection);
+        } else {
+            addAccount(selection);
+        }
+    };
+
+    const chipGroupComponent = () => {
+        return (
+            <LabelGroup numLabels={10}>
+                {(selected || []).map((currentLabel, index) => {
+                    const optional = !current_group.userlistPrimary.includes(currentLabel) && superuser.allowed ? { onClose: () => removeAccount(currentLabel) } : {};
+
+                    return (
+                        <Label key={currentLabel}
+                               color={accounts.find(ac => ac.name === currentLabel).isAdmin ? "gold" : "cyan"}
+                               {...optional}
+                        >
+                            {currentLabel}
+                        </Label>
+                    );
+                })}
+            </LabelGroup>
+        );
+    };
+
+    return (
+        <FormGroup
+            fieldId="group-accounts"
+            helperText={
+                (history.length > 0)
+                    ? <HelperText className="pf-c-form__helper-text">
+                        <Flex>
+                            {loggedInUsersAffected && <HelperTextItem id="group-accounts-helper" variant="warning">{_("To take effect, the changed users must log out.")}</HelperTextItem>}
+                            {history.length > 0 && <Button variant="link" id="group-undo-btn" isInline icon={<UndoIcon />} onClick={undoAccountChanges}>{_("Undo")}</Button>}
+                        </Flex>
+                    </HelperText>
+                    : ''
+            }
+            id="group-accounts-form-group"
+            label={_("Accounts")}
+            validated={history.length > 0 ? "warning" : "default"}
+        >
+            {superuser.allowed
+                ? <Select
+                   chipGroupComponent={chipGroupComponent()}
+                   isDisabled={!superuser.allowed}
+                   isOpen={isOpenAccount}
+                   onSelect={onSelectAccount}
+                   onToggle={setIsOpenAccount}
+                   selections={selected}
+                   toggleId="group-accounts"
+                   variant={SelectVariant.typeaheadMulti}
+                >
+                    {accounts.map((option, index) => (
+                        <SelectOption
+                            isDisabled={current_group.userlistPrimary.includes(option.name)}
+                            key={index}
+                            value={option.name}
+                        />
+                    ))}
+                </Select>
+                : chipGroupComponent()}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230309-102831-8629c1ab-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)